INSTALL.md

```diff
@@ -122,10 +119,15 @@ Ginkgo adds the following additional switches to control what is being built:
     [`ARCHITECTURES` specification list](https://github.com/ginkgo-project/CudaArchitectureSelector/blob/master/CudaArchitectureSelector.cmake#L58)
     section in the documentation of the CudaArchitectureSelector CMake module.
 
+Additionally, the following CMake options have effect on the build process:
+
+*  `-DCMAKE_EXPORT_PACKAGE_REGISTRY={ON,OFF}` if set to `ON` the build directory will
+   be stored in the current user's CMake package registry.
+
 For example, to build everything (in debug mode), use:
 
 ```cmake
-cmake  -G "Unix Makefiles" -H. -BDebug -DCMAKE_BUILD_TYPE=Debug -DGINKGO_DEVEL_TOOLS=ON \
+cmake -S. -BDebug -DCMAKE_BUILD_TYPE=Debug -DGINKGO_DEVEL_TOOLS=ON \
```

Comment:
```suggestion
cmake -S.. -BDebug -DCMAKE_BUILD_TYPE=Debug -DGINKGO_DEVEL_TOOLS=ON \
```
Maybe keep using the build (or any child folder) not the source directory as the build directory